Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiManager.cpp

```diff
@@ -266,6 +266,16 @@ namespace AzToolsFramework::ViewportUi
         m_viewportUi->CreateViewportBorder(borderTitle, backButtonCallback);
     }
 
+    bool ViewportUiManager::GetViewportBorderVisible() const
+    {
+        return m_viewportUi->GetViewportBorderVisible();
+    }
+
+    void ViewportUiManager::ChangeViewportBorderText(const AZStd::string borderTitle)
```

Comment:
update: Please can this be a `const AZStd::string&` (so we're not making a copy here)